config/application.rb

```diff
@@ -70,8 +70,8 @@ class Application < Rails::Application
     config.autoload_paths << "#{Rails.root.to_s}/lib/health_checks"
 
     # See Rails::Configuration for more options
-    ENV['RECAPTCHA_PUBLIC_KEY'] = ::AlaveteliConfiguration::recaptcha_public_key
-    ENV['RECAPTCHA_PRIVATE_KEY'] = ::AlaveteliConfiguration::recaptcha_private_key
+    ENV['RECAPTCHA_SITE_KEY'] = AlaveteliConfiguration.recaptcha_site_key
+    ENV['RECAPTCHA_SECRET_KEY'] = AlaveteliConfiguration.recaptcha_secret_key
```

Comment:
Squash 2684989e5261f30c170ac1600add735dda997f80 in to this commit.

+1 on renaming; easier to figure out what to google to get the keys / fix issues etc.